isubit/luggage_resources
# LUGG-828 Universal Grid Styling

See also: https://github.com/isubit/suitcase/pull/52
## Changes
- Removed grid styling luggage_resources and moved it to suitcase.css
- Removed hook_views_preprocess() that previously controlled the resource grid styling
## Testing
- Add some resources
- Pull down these changes
- Make sure that you don’t have the luggage_resources feature locked
- Clear caches and reset all features (I also run the drush command registry-rebuild, but that might be overkill)
- Go to the resources view. Is there still a nice resources grid?
- Resize the window, make sure that each resource cell resizes as you change the window size. Is any text cut off or does text overflow the grid cell? Is the resource link still green?
- Go to a resources node. Is it still styled?
- Again, add a resource. Is the node creation form still styled?
